stdlib/2and3/decimal.pyi

```diff
@@ -49,24 +55,37 @@ class Overflow(Inexact, Rounded): ...
 
 class Underflow(Inexact, Rounded, Subnormal): ...
 
-def setcontext(context: Context): ...
+if sys.version_info >= (3,):
+    class FloatOperation(DecimalException, TypeError): ...
+
+def setcontext(context: Context) -> None: ...
 def getcontext() -> Context: ...
 def localcontext(ctx: Optional[Context] = ...) -> _ContextManager: ...
 
-class Decimal(SupportsAbs[Decimal], SupportsFloat, SupportsInt):
-    def __init__(cls, value: Union[_Decimal, float, str, unicode,
+if sys.version_info >= (3,):
+    _SupportsRound = SupportsRound[int]
+else:
+    _SupportsRound = object
+
+class Decimal(SupportsAbs[Decimal], SupportsFloat, SupportsInt, _SupportsRound):
+    # TODO: SupportsCeil, SupportsFloor, SupportsTrunc?
+
+    def __init__(cls, value: Union[_Decimal, float, Text,
                                    Tuple[int, Sequence[int], int]] = ...,
                  context: Context = ...) -> None: ...
     @classmethod
     def from_float(cls, f: float) -> Decimal: ...
-    def __nonzero__(self) -> bool: ...
+    if sys.version_info >= (3,):
+        def __bool__(self) -> bool: ...
+    else:
+        def __nonzero__(self) -> bool: ...
     def __eq__(self, other: object) -> bool: ...
     def __ne__(self, other: object) -> bool: ...
     def __lt__(self, other: _ComparableNum) -> bool: ...
     def __le__(self, other: _ComparableNum) -> bool: ...
     def __gt__(self, other: _ComparableNum) -> bool: ...
     def __ge__(self, other: _ComparableNum) -> bool: ...
-    def compare(self, other: _Decimal) -> Decimal: ...
+    def compare(self, other: _Decimal, context: Optional[Context] = ...) -> Decimal: ...
     def __hash__(self) -> int: ...
     def as_tuple(self) -> DecimalTuple: ...
```

Comment:
Missing `as_integer_ratio`, which is `def as_integer_ratio(self) -> Tuple[int, int]`. (Haven't checked yet whether it exists in Python 2.)